webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9628,6 +9627,36 @@ var userOverride = {
                 return '';
             };
 
+            
+            _this.getMessageCreatedAtTime = function (createdAtTime) {
+                if (TIME_FORMAT_24HOURS) {
+                    let month = ["Jan","Feb","March","April","May","June","July","Aug","Sep","Oct","Nov","Dec"];
+                    var msgtime = new Date(createdAtTime);
+                    let messageDay = msgtime.getDate(createdAtTime);
+                    let monthName = month[msgtime.getMonth(createdAtTime)];
+                    function addZero(i) {
+                        if (i < 10) {
+                            return "0" + i;
+                        } else {
+                            return i;
+                        }
+                    }
+                    var hours = addZero(msgtime.getHours());
+                    var minute = addZero(msgtime.getMinutes());
+                    if(msgtime.getHours()<24){
+                        var time =   hours + ":" + minute;
+                    }else{
+                        var time = monthName + " " + messageDay + ", " + hours + ":" + minute;
```

Comment:
as it used below, it can't be a local variable inside if else

Reply:
made changes